src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,373 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True">
+  <Design.PreviewWith>
+  </Design.PreviewWith>
+
+  <Styles.Resources>
+    <x:Double x:Key="SplitButtonPrimaryButtonSize">32</x:Double>
+    <x:Double x:Key="SplitButtonSecondaryButtonSize">32</x:Double>
+  </Styles.Resources>
+
+  <Style Selector="SplitButton">
+    <Setter Property="Background" Value="{DynamicResource SplitButtonBackground}" />
+    <Setter Property="Foreground" Value="{DynamicResource SplitButtonForeground}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource SplitButtonBorderBrush}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource SplitButtonBorderThemeThickness}" />
+    <Setter Property="HorizontalAlignment" Value="Left" />
+    <Setter Property="VerticalAlignment" Value="Center" />
+    <!--<Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+    <Setter Property="FontWeight" Value="Normal" />
+    <Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}" />
+    <Setter Property="UseSystemFocusVisuals" Value="True" />
+    <Setter Property="FocusVisualMargin" Value="-3" />-->
+    <Setter Property="KeyboardNavigation.IsTabStop" Value="True" />
+    <Setter Property="Padding" Value="{DynamicResource ButtonPadding}" />
+    <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate>
+          <!-- A Border was added here since Avalonia does not support Background/Border/CornerRadius on Grid directly -->
+          <!-- ClipToBounds added as a temporary work-around for: https://github.com/AvaloniaUI/Avalonia/issues/2105 -->
+          <Border Background="Transparent"
+                  CornerRadius="{TemplateBinding CornerRadius}"
+                  ClipToBounds="True">
```

Comment:
@maxkatz6 This ClipToBounds is required or there are no rounded corners in the control. This is what I was referring to in the discussion here: https://github.com/AvaloniaUI/Avalonia/issues/2105.

It's possible I misunderstand something.

Reply:
In Expander fluent style we have set corner radius separately on internal components. Instead of having common parent cropping children. It kind of makes more sense for me even aside from that issue you mentioned. Though I don't have strong opinion here and it might be easier in this way.

Reply:
Ah, well to do that like Expander you would need special converters to select which corners of the corner radius apply to which part. (The SplitButton is composed of two child buttons in between them there should be no corners). That is what was done for Expander and I recall your discussion about the converters so it is possible -- it seems much less simple than just setting ClipToBounds though.

Reply:
I think Fluent V2 adds Borders to SplitButton. This means the ClipToBounds work-around wouldn't work as it would hack off the edges that need to show borders. I think I will re-work this to use converters and do what Expander does. Otherwise, it's just more work when Fluent v2 is applied and we have to re-learn this conversation.

Reply:
Resolved: I removed the ClipToBounds 'hack' and now set the CornerRadius using the same converters in Expander for all relevant child controls.